docs/admin.rst

```diff
@@ -294,6 +294,36 @@ web server to apply the changes:
 
   sudo service apache2 restart
 
+.. _submission-cleanup:
+
+Cleaning up deleted submissions
+&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+When submissions are deleted through the web interface, their database
+records are deleted and their files are securely wiped. For large
+files, secure removal can take some time, and it's possible, though
+unlikely, that it can be interrupted, for example by a server
+reboot. In older versions of SecureDrop this could leave a submission
+file present without a database record.
+
+As of SecureDrop 1.0.0, automated checks send OSSEC alerts when this
+situation is detected, recommending you run ``manage.py
```

Comment:
IMO we should describe here again how to run this command.

1) Make it explicit that the admin needs to SSH into the app server

2) Make it explicit that they need to run

```
sudo -u www-data bash
cd /var/www/securedrop
```
... and then have the `manage.py` commands prefixed with `./`

Reply:
Also , having these within a code block like
```
.. code:: sh
  commands here
```

Reply:
Done.